.github/workflows/workflow.yaml

```diff
@@ -150,6 +150,39 @@ jobs:
           path: |
             logs-*.json
 
+  build_mutation_test:
```

Comment:
TODO for later: `build_mutation_test` will need to be added to `needs` for release and pre-release once we are ready